portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -73,13 +77,19 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     
-    List<String> commentsList = new ArrayList();
+    List<Comment> commentsList = new ArrayList();
 
     Query commentsQuery = new Query("Comment");
     PreparedQuery results = datastore.prepare(commentsQuery);
 
     for (Entity entity : results.asIterable()){
-      commentsList.add((String)entity.getProperty("comment_msg"));
+
+      Comment cmt = new Comment(
+        (String)entity.getProperty("name"),
```

Comment:
nit: add a space between `(String)` and `entity.getProperty`

Reply:
Done.